pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -727,6 +760,63 @@ func TestTaskValidateError(t *testing.T) {
 	}
 }
 
+func TestTaskValidateErrorWithStepActionRef(t *testing.T) {
```

Comment:
I'm not sure if this test is really necessary? Seems like the code is already covered by the `TestTaskiSpecValidateErrorWithStepActionRef` test below, and this looks like a duplicate

Reply:
good point. I will remove it.

Reply:
done.